src/gateways/BeaconsApiAccountHolderGateway.ts

```diff
@@ -29,13 +30,14 @@ export class BeaconsApiAccountHolderGateway implements AccountHolderGateway {
       >(url, {
         headers: { Authorization: `Bearer ${await this.getAccessToken()}` },
       });
+      logger.info("Account holder id retrieved");
       return response.data.id;
     } catch (error) {
       if (error.response && error.response.status === 404) {
         return null; // 404 is a-ok
       }
       /* eslint-disable no-console */
```

Comment:
Does eslint get upset with `logger.info` statements as well as `console.log`?  If eslint is happy, then could you remove the `/* eslint-disable no-console */` above logging statements?